resources/assets/components/Signup/index.js

```diff
@@ -78,12 +78,23 @@ class Signup extends React.Component {
   }
 
   // Open the history modal of the given post
-  showHistory(postId, event) {
+  showHistory(postId, event, signupId) {
```

Comment:
It looks like this component has a `signup_id` prop, is that safe to use here? Or if not, can we get the signup ID from the given `postId` argument?

Reply:
so right!! thanks for catching that !